src/c++/vdo/base/message-stats.c

```diff
@@ -11,749 +11,407 @@
 	/* The total number of items for which a write operation has completed */
-	result = write_u64("committed : ", stats->committed, ", ", buf, maxlen);
-	if (result != VDO_SUCCESS)
-		return result;
-	result = write_string(NULL, "}", suffix, buf, maxlen);
-	if (result != VDO_SUCCESS)
-		return result;
-	return VDO_SUCCESS;
+	write_u64("committed : ", stats->committed, ", ", buf, maxlen);
+	write_string(NULL, "}", suffix, buf, maxlen);
 }
 
-static int write_recovery_journal_statistics(char *prefix,
-					     struct recovery_journal_statistics *stats,
-					     char *suffix, char **buf,
-					     unsigned int *maxlen)
+static void write_recovery_journal_statistics(char *prefix,
+					      struct recovery_journal_statistics *stats,
+					      char *suffix, char **buf,
+					      unsigned int *maxlen)
 {
-	int result;
-
-	result = write_string(prefix, "{ ", NULL, buf, maxlen);
-	if (result != VDO_SUCCESS)
-		return result;
+	write_string(prefix, "{ ", NULL, buf, maxlen);
 	/* Number of times the on-disk journal was full */
-	result = write_u64("diskFull : ", stats->disk_full, ", ", buf, maxlen);
-	if (result != VDO_SUCCESS)
-		return result;
+	write_u64("diskFull : ", stats->disk_full, ", ", buf, maxlen);
 	/* Number of times the recovery journal requested slab journal commits. */
-	result = write_u64("slabJournalCommitsRequested : ",
-			   stats->slab_journal_commits_requested, ", ", buf, maxlen);
-	if (result != VDO_SUCCESS)
-		return result;
+	write_u64("slabJournalCommitsRequested : ",
+		  stats->slab_journal_commits_requested, ", ", buf, maxlen);
 	/* Write/Commit totals for individual journal entries */
-	result = write_commit_statistics("entries : ", &stats->entries, ", ", buf,
-					 maxlen);
-	if (result != VDO_SUCCESS)
-		return result;
+	write_commit_statistics("entries : ", &stats->entries, ", ", buf, maxlen);
 	/* Write/Commit totals for journal blocks */
-	result = write_commit_statistics("blocks : ", &stats->blocks, ", ", buf, maxlen);
-	if (result != VDO_SUCCESS)
-		return result;
-	result = write_string(NULL, "}", suffix, buf, maxlen);
-	if (result != VDO_SUCCESS)
-		return result;
-	return VDO_SUCCESS;
+	write_commit_statistics("blocks : ", &stats->blocks, ", ", buf, maxlen);
+	write_string(NULL, "}", suffix, buf, maxlen);
 }
 
-static int write_packer_statistics(char *prefix, struct packer_statistics *stats,
-				   char *suffix, char **buf, unsigned int *maxlen)
+static void write_packer_statistics(char *prefix, struct packer_statistics *stats,
+				    char *suffix, char **buf, unsigned int *maxlen)
 {
-	int result;
-
-	result = write_string(prefix, "{ ", NULL, buf, maxlen);
-	if (result != VDO_SUCCESS)
-		return result;
+	write_string(prefix, "{ ", NULL, buf, maxlen);
 	/* Number of compressed data items written since startup */
-	result = write_u64("compressedFragmentsWritten : ",
+	write_u64("compressedFragmentsWritten : ",
 			   stats->compressed_fragments_written, ", ", buf, maxlen);
```

Comment:
Indentation of the continuation line needs adjusting

Reply:
I have made the changes.